pydm/widgets/analog_indicator.py

```diff
@@ -0,0 +1,1576 @@
+from PyQt5.QtCore import QPointF
+
+from .base import PyDMWidget, TextFormatter
+from qtpy.QtGui import QColor, QPolygon, QPen, QPainter
+from qtpy.QtWidgets import QFrame, QVBoxLayout, QHBoxLayout, QLabel, QSizePolicy, QWidget, QGridLayout
+from qtpy.QtCore import Qt, QPoint, Property
+from qtpy.QtWidgets import QWIDGETSIZE_MAX
+from .channel import PyDMChannel
+import sys
+
+
+class QScale(QFrame):
```

Comment:
Are you using the `QFrame` inheritance to let the user adjust the frame style? I don't see anything in the `QScale` class specifically that references it, so it isn't obvious to me why this isn't just a `QWidget`.